pkg/launcher/chrome_profile.go

```diff
@@ -0,0 +1,35 @@
+package launcher
+
+import (
+	"fmt"
+	"hash/fnv"
+)
+
+type ChromeProfile struct {
+	// ExecutablePath is the path to the Chrome binary on the system.
+	ExecutablePath string
+	// UserDataPath is the path to the Chrome user data directory,
+	// which we override to put Granted profiles in a specific folder
+	// for easy management.
+	UserDataPath string
+}
+
+func (l ChromeProfile) LaunchCommand(url string, profile string) []string {
+	profileName := chromeProfileName(profile)
+	return []string{
+		l.ExecutablePath,
+		"--user-data-dir=" + l.UserDataPath,
+		"--profile-directory=" + profileName,
+		"--no-first-run",
+		"--no-default-browser-check",
+		url,
+	}
+}
+
+func chromeProfileName(profile string) string {
```

Comment:
@meyerjrr I've kept this as-is from the previous implementation. Is there any reason we are using a hash, rather than just the name of the profile itself here?

Reply:
Was to help with the container functionality from memory, there was an issue with overlapping names. Will test it out

Reply:
No worries, I've left it all the same as previous to avoid breaking changes. In future it could be good to see if we could just use the profile name here